### PR TITLE
[Merged by Bors] - chore(LpSpace): golf

### PR DESCRIPTION
**Pull request by urkud**
Cherry-picked from #6839

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Function/LpSpace.lean [hunk 1]
```diff
@@ -1001,10 +1000,9 @@ variable {g : E → F} {c : ℝ≥0}
 theorem LipschitzWith.comp_memℒp {α E F} {K} [MeasurableSpace α] {μ : Measure α}
     [NormedAddCommGroup E] [NormedAddCommGroup F] {f : α → E} {g : E → F} (hg : LipschitzWith K g)
     (g0 : g 0 = 0) (hL : Memℒp f p μ) : Memℒp (g ∘ f) p μ :=
-  haveI : ∀ x, ‖g (f x)‖ ≤ K * ‖f x‖ := by
-    intro a
+  have : ∀ x, ‖g (f x)‖ ≤ K * ‖f x‖ := fun x ↦ by
     -- TODO: add `LipschitzWith.nnnorm_sub_le` and `LipschitzWith.nnnorm_le`
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Function/LpSpace.lean:1004]: Did I end up adding these, or is that stuck in a mathlib3 PR?

File: Mathlib/MeasureTheory/Function/LpSpace.lean [hunk 2]
```diff
@@ -270,6 +269,10 @@ protected theorem coe_nnnorm (f : Lp E p μ) : (‖f‖₊ : ℝ) = ‖f‖ :=
   rfl
 #align measure_theory.Lp.coe_nnnorm MeasureTheory.Lp.coe_nnnorm
 
+@[simp, norm_cast]
+theorem nnnorm_coe_ennreal (f : Lp E p μ) : (‖f‖₊ : ℝ≥0∞) = snorm f p μ :=
+  ENNReal.coe_toNNReal <| ne_of_lt f.2
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Function/LpSpace.lean:274]:
```suggestion
  ENNReal.coe_toNNReal f.snorm_lt_top.ne
```

**Review by eric-wieser — APPROVED** [review]
bors d+

File: Mathlib/MeasureTheory/Function/LpSpace.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/MeasureTheory/Function/LpSpace.lean:1004]: API doc search: not found.

File: Mathlib/MeasureTheory/Function/LpSpace.lean [hunk 3]
```diff
@@ -270,6 +269,10 @@ protected theorem coe_nnnorm (f : Lp E p μ) : (‖f‖₊ : ℝ) = ‖f‖ :=
   rfl
 #align measure_theory.Lp.coe_nnnorm MeasureTheory.Lp.coe_nnnorm
 
+@[simp, norm_cast]
+theorem nnnorm_coe_ennreal (f : Lp E p μ) : (‖f‖₊ : ℝ≥0∞) = snorm f p μ :=
+  ENNReal.coe_toNNReal f.snorm_ne_top
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Function/LpSpace.lean:274]: Ugh, sorry; I guess it's `<| Lp.norm_ne_top f`

Comment (urkud) [reply Mathlib/MeasureTheory/Function/LpSpace.lean:274]:
```suggestion
  ENNReal.coe_toNNReal <| Lp.snorm_ne_top f
```

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**